modules/ob-about-build-controller.adoc

```diff
@@ -4,14 +4,14 @@
 
 :_content-type: CONCEPT
 [id="about-build-controller_{context}"]
-= About the build controller
+= Build controller
```

Comment:
The wording seems too general here. When scanning the page, the user won't necessarily know what info the section contains. Consider revising this heading to a more descriptive phrase such as "Build controller features" or "Build controller overview"

Reply:
@bburt-rh, The assembly title is Overview of OpenShift Builds and we are discussing key custom resources and the build controller component. Right now, we have a parallel structure among headings and the word `controller` itself is an entity. Even if we add `overview` or `features` in the end, it does not make much of a difference and is clear from the assembly title that all the modules is part of overview.